jenkins/packer/llm-studio-cloud-images.groovy

```diff
@@ -0,0 +1,104 @@
+import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
+
+properties(
+    [
+        parameters(
+            [
+                string(name: 'BRANCH_TAG', defaultValue: 'origin/main'),
+                booleanParam(name: 'AWS', defaultValue: true, description: 'Make Amazon Machine Image/Not?'),
+                booleanParam(name: 'GCP', defaultValue: true, description: 'Make GCP Image/Not?'),
+                booleanParam(name: 'AZURE', defaultValue: true, description: 'Make AZURE Image/Not?'),
+                string(name: 'LLM_STUDIO_VERSION')
```

Comment:
can you add a description with a example following the correct format

Reply:
sure